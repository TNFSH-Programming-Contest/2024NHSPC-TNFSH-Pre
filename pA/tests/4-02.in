728837164018669019 482392994221327973 480797126235279083
480797126235279083 728837164018669019 482392994221327973
