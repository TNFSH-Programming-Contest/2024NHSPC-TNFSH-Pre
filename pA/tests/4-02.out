1197543076894545093
